### PR TITLE
Closes 1839_dummy_issue_onboarding_for_im

### PR DESCRIPTION
**Pull request by jimrothstein**
`First PR .... not quite finished,  especially formatting` 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Review the [Cheat Sheet](https://github.com/pharmaverse/admiral/blob/main/inst/cheatsheet/admiral_cheatsheet.pdf). Make any required updates to it by editing the file `inst/cheatsheet/admiral_cheatsheet.pptx` and re-upload a PDF version of it to the same xfolder.
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers). A Developer Notes section is available in `NEWS.md` for tracking developer-facing issues.  
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: tests/testthat/test-hello_admiral.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+S = "Welcome to Admiral family"
```
Comment (bms63) [inline tests/testthat/test-hello_admiral.R:2]: We are trying to move data into test_that calls so there is no spillover issues between test files.

**Comment by bms63** [comment]
Change title of PR to Closes #1839...

**Comment by bms63** [comment]
Just do devtools::document to get `.Rd` files and checks should pass.

**Comment by jimrothstein** [comment]
@bms63   Issue #1839 
kk
Tkkhx for review comments.
-  Moved test data (String S) into test_that function.
- Documented again.
- Fixed title "Closed issue ...."
Hope better now.

File: tests/testthat/test-hello_admiral.R (hunk 1 above)
Comment (jimrothstein) [reply tests/testthat/test-hello_admiral.R]: Made this change.

**Comment by bms63** [comment]
* Do a lintr::lint_package() or lintr::lint_file(your_files) and the same using styler

* I don't see the `.Rd` files in the PR.  Are they being created? If so, you need to push those up.

**Comment by jimrothstein** [comment]
Lame excuse:   Switching from neovim to emacs for R.

**Comment by bms63** [comment]
Looks like just a couple more minor checks to fix!!

* Lintr and Styler need to be run and then changes pushed up
* Run spelling check - see https://docs.ropensci.org/spelling/
* Looks like the "Closes..." Title didn't get saved

**Comment by manciniedoardo** [comment]
@jimrothstein almost perfect now, just need to add the words flagged by the spellcheck to `inst/WORDLIST` 😄

**Comment by jimrothstein** [comment]
Confused.   Ran spelling check several ways;  with `spelling::` and `devtools::`.   Reports no spelling errors.   But RStudio still showing a few phrases 'hw', 'der_*'  as spelling errors.    and words NOT in WORDLIST.
Doesn't work if I manually add the words, or use spelling::update_wordlist()

**Comment by manciniedoardo** [comment]
@jimrothstein hmmm, looks like it passes now anyway! maybe it was just a transient issue

**Comment by manciniedoardo** [comment]
@jimrothstein can you please close this PR? thanks

**Pull request closed without merging**